lib/actions/api.js

```diff
@@ -443,6 +443,13 @@ export function createQueryAction(
   return async function (dispatch, getState) {
     const state = getState()
     const { config } = state.otp
+
+    if (options.appendBounds) {
+      const bounds = config?.geocoder?.boundary?.rect
+      const stringifiedBounds = '&' + qs.stringify(bounds)
```

Comment:
Should there be a check for null `bounds`?

Reply:
Good catch thanks! Addressed in 4ddf2c1901d354aea38d19c23a38edb8b0de2eab